--- origsrc/cjose-0.6.2.8/test/Makefile.am	2026-08-12 12:52:55.000000000 +0000
+++ src/cjose-0.6.2.8/test/Makefile.am	2026-10-06 00:44:28.827548000 +0000
@@ -7,7 +7,9 @@ TESTS = check_cjose
 
 check_PROGRAMS = $(TESTS)
 check_cjose_CFLAGS = @CHECK_CFLAGS@ -I$(top_builddir)/include \
-                     -I$(top_builddir)/src -I$(top_srcdir)/src
+                     -I$(top_builddir)/src -I$(top_srcdir)/src \
+                     -I$(top_builddir)/include -I$(top_srcdir)/include \
+                     -I$(top_builddir)/include/cjose
 check_cjose_LDADD = $(top_builddir)/src/libcjose.la @CHECK_LIBS@
 check_cjose_SOURCES = check_cjose.c \
                       check_version.c \
