--- origsrc/oneTBB-2023.1.0/cmake/compilers/GNU.cmake	2026-07-10 18:07:17.000000000 +0000
+++ src/oneTBB-2023.1.0/cmake/compilers/GNU.cmake	2026-10-05 21:17:36.836196200 +0000
@@ -114,7 +114,7 @@ endif ()
 set(TBB_LIB_COMPILE_FLAGS ${TBB_LIB_COMPILE_FLAGS} $<$<NOT:$<VERSION_LESS:${CMAKE_CXX_COMPILER_VERSION},8.0>>:-fstack-clash-protection>)
 
 # -z switch is not supported on MacOS and MinGW
-if (NOT APPLE AND NOT MINGW)
+if (NOT APPLE AND NOT MINGW AND NOT CYGWIN)
     set(TBB_LIB_LINK_FLAGS ${TBB_LIB_LINK_FLAGS} -Wl,-z,relro,-z,now,-z,noexecstack)
 endif()
 if (NOT CMAKE_CXX_FLAGS MATCHES "_FORTIFY_SOURCE")
--- origsrc/oneTBB-2023.1.0/include/oneapi/tbb/detail/_config.h	2026-07-10 18:07:17.000000000 +0000
+++ src/oneTBB-2023.1.0/include/oneapi/tbb/detail/_config.h	2026-10-05 21:17:36.838222600 +0000
@@ -208,7 +208,7 @@
 
 /** __TBB_WEAK_SYMBOLS_PRESENT denotes that the system supports the weak symbol mechanism **/
 #ifndef __TBB_WEAK_SYMBOLS_PRESENT
-    #define __TBB_WEAK_SYMBOLS_PRESENT ( !__EMSCRIPTEN__ && !_WIN32 && !__APPLE__ && !__sun && (__TBB_GCC_VERSION >= 40000 || __INTEL_COMPILER ) )
+    #define __TBB_WEAK_SYMBOLS_PRESENT ( !__EMSCRIPTEN__ && !_WIN32 && !__CYGWIN__ && !__APPLE__ && !__sun && (__TBB_GCC_VERSION >= 40000 || __INTEL_COMPILER ) )
 #endif
 
 /** Presence of compiler features **/
--- origsrc/oneTBB-2023.1.0/test/common/common_arena_constraints.h	2026-07-10 18:07:17.000000000 +0000
+++ src/oneTBB-2023.1.0/test/common/common_arena_constraints.h	2026-10-05 21:17:36.838222600 +0000
@@ -91,7 +91,7 @@ int get_processors_group_count() { retur
 #define __HWLOC_TOPOLOGY_FLAG_RESTRICT_TO_CPUBINDING_PRESENT (HWLOC_API_VERSION >= 0x20500)
 // At this moment the hybrid CPUs HWLOC interfaces returns unexpected results on some Windows machines
 // in the 32-bit arch mode.
-#define __HWLOC_HYBRID_CPUS_INTERFACES_VALID (!_WIN32 || _WIN64)
+#define __HWLOC_HYBRID_CPUS_INTERFACES_VALID (!_WIN32)
 
 #define __HYBRID_CPUS_TESTING __HWLOC_HYBRID_CPUS_INTERFACES_PRESENT && __HWLOC_HYBRID_CPUS_INTERFACES_VALID
 #define __HWLOC_CPUBIND_PRESENT (!__APPLE__ && !__gnu_hurd__)
--- origsrc/oneTBB-2023.1.0/test/tbb/test_eh_thread.cpp	2026-07-10 18:07:17.000000000 +0000
+++ src/oneTBB-2023.1.0/test/tbb/test_eh_thread.cpp	2026-10-05 21:17:36.840244100 +0000
@@ -35,7 +35,7 @@
 // On Windows there is no real thread number limit beside available memory.
 // Therefore, the test for thread limit is unreasonable.
 // TODO: enable limitThreads with sanitizer under docker
-#if TBB_USE_EXCEPTIONS && !_WIN32 && !__ANDROID__
+#if TBB_USE_EXCEPTIONS && !_WIN32 && !__ANDROID__ && !__CYGWIN__
 
 #include <limits.h>
 #include <sys/types.h>
