--- origsrc/aws-lambda-cpp-1.0.1/CMakeLists.txt	2026-08-19 00:00:48.000000000 +0900
+++ src/aws-lambda-cpp-1.0.1/CMakeLists.txt	2026-10-03 09:31:47.831725000 +0900
@@ -1,5 +1,4 @@
 cmake_minimum_required(VERSION 3.9)
-set(CMAKE_CXX_STANDARD 11)
 set(AWS_LAMBDA_CPP_VERSION "0.0.0" CACHE STRING
     "Library version, injected at release time from the git tag.")
 project(aws-lambda-runtime
@@ -142,13 +141,19 @@ configure_file("${CMAKE_CURRENT_SOURCE_D
 export(EXPORT "${PROJECT_NAME}-targets" NAMESPACE AWS::)
 
 install(EXPORT "${PROJECT_NAME}-targets"
-    DESTINATION "${CMAKE_INSTALL_LIBDIR}/${PROJECT_NAME}/cmake/"
+    DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}/"
     NAMESPACE AWS::)
 
 install(FILES "${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}-config.cmake"
     "${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}-config-version.cmake"
-    DESTINATION "${CMAKE_INSTALL_LIBDIR}/${PROJECT_NAME}/cmake/")
+    DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}/")
 
 install(PROGRAMS "${CMAKE_CURRENT_SOURCE_DIR}/packaging/packager"
-    DESTINATION "${CMAKE_INSTALL_LIBDIR}/${PROJECT_NAME}/cmake/")
+    DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}/")
 
+set(prefix ${CMAKE_INSTALL_PREFIX})
+set(exec_prefix ${CMAKE_INSTALL_PREFIX})
+set(libdir ${CMAKE_INSTALL_PREFIX}/lib)
+set(includedir ${CMAKE_INSTALL_PREFIX}/include)
+configure_file(${CMAKE_SOURCE_DIR}/aws-lambda-cpp.pc.in ${CMAKE_BINARY_DIR}/aws-lambda-cpp.pc @ONLY)
+install(FILES ${CMAKE_BINARY_DIR}/aws-lambda-cpp.pc DESTINATION ${CMAKE_INSTALL_PREFIX}/lib/pkgconfig)
--- origsrc/aws-lambda-cpp-1.0.1/aws-lambda-cpp.pc.in	1970-01-01 09:00:00.000000000 +0900
+++ src/aws-lambda-cpp-1.0.1/aws-lambda-cpp.pc.in	2026-10-03 09:33:10.037614600 +0900
@@ -0,0 +1,10 @@
+prefix=@prefix@
+exec_prefix=@exec_prefix@
+libdir=@libdir@
+includedir=@includedir@
+
+Name: aws-lambda-cpp
+Description: C++ implementation of the AWS Lambda runtime
+Version: @VERSION@
+Libs: -L${libdir} -laws-lambda-runtime
+Cflags: -I${includedir}
--- origsrc/aws-lambda-cpp-1.0.1/tests/CMakeLists.txt	2026-08-19 00:00:48.000000000 +0900
+++ src/aws-lambda-cpp-1.0.1/tests/CMakeLists.txt	2026-10-03 09:17:39.844291300 +0900
@@ -1,25 +1,22 @@
 cmake_minimum_required(VERSION 3.11)
 project(aws-lambda-runtime-tests LANGUAGES CXX)
 
-include(FetchContent)
-FetchContent_Declare(gtest
-    URL https://github.com/google/googletest/archive/v1.12.0.tar.gz
-)
-set(gtest_force_shared_crt ON CACHE BOOL "" FORCE)
-set(BUILD_GMOCK OFF CACHE BOOL "" FORCE)
-set(INSTALL_GTEST OFF)
-FetchContent_MakeAvailable(gtest)
+find_package(gtest)
+if (NOT GTEST_FOUND)
+    include(FetchContent)
+    FetchContent_Declare(gtest
+        URL https://github.com/google/googletest/archive/v1.12.0.tar.gz
+    )
+    set(gtest_force_shared_crt ON CACHE BOOL "" FORCE)
+    set(BUILD_GMOCK OFF CACHE BOOL "" FORCE)
+    set(INSTALL_GTEST OFF)
+    FetchContent_MakeAvailable(gtest)
+endif()
 
 add_executable(unit_tests
     unit/thread_local_curl_test.cpp
     unit/unit_tests.cpp)
-target_link_libraries(unit_tests PRIVATE gtest_main aws-lambda-runtime)
-
-include(GoogleTest)
-gtest_discover_tests(unit_tests
-    PROPERTIES
-    LABELS "unit"
-    DISCOVERY_TIMEOUT 10)
+target_link_libraries(unit_tests PRIVATE gtest gtest_main aws-lambda-runtime)
 
 add_subdirectory(resources)
 
