--- origsrc/libcpuid-0.8.2/libcpuid/Makefile.am	2026-09-13 17:44:30.000000000 +0000
+++ src/libcpuid-0.8.2/libcpuid/Makefile.am	2026-10-09 23:35:49.339614300 +0000
@@ -46,7 +46,7 @@ noinst_HEADERS = 		\
 EXTRA_DIST += libcpuid.sym libcpuid_vc71.vcproj libcpuid_vc10.vcxproj libcpuid_vc10.vcxproj.filters
 
 if HAVE_DOXYGEN
-directory = $(top_srcdir)/libcpuid/docs/man/man3
+directory = $(top_builddir)/libcpuid/docs/man/man3
 
 dist_man_MANS = $(directory)/cpu_id_t.3		\
 	$(directory)/cpu_list_t.3		\
