--- origsrc/libBigWig-0.4.9/CMakeLists.txt	2026-09-14 10:24:26.000000000 +0000
+++ src/libBigWig-0.4.9/CMakeLists.txt	2026-10-09 23:14:17.786930900 +0000
@@ -51,6 +51,10 @@ set(LIBBIGWIG_PUBLIC_HEADERS "bigWig.h;b
 set_target_properties(BigWig PROPERTIES PUBLIC_HEADER
                                         "${LIBBIGWIG_PUBLIC_HEADERS}")
 
+if (DEFINED SOVERSION)
+  set_target_properties(BigWig PROPERTIES SOVERSION "${SOVERSION}")
+endif()
+
 if(ENABLE_TESTING)
   add_subdirectory(test)
 endif()
@@ -63,3 +67,10 @@ install(
   LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
   ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
   PUBLIC_HEADER DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/libbigwig)
+
+set(prefix ${CMAKE_INSTALL_PREFIX})
+set(exec_prefix ${CMAKE_INSTALL_PREFIX})
+set(libdir ${CMAKE_INSTALL_PREFIX}/lib)
+set(includedir ${CMAKE_INSTALL_PREFIX}/include)
+configure_file(${CMAKE_SOURCE_DIR}/libBigWig.pc.in ${CMAKE_BINARY_DIR}/libBigWig.pc @ONLY)
+install(FILES ${CMAKE_BINARY_DIR}/libBigWig.pc DESTINATION ${CMAKE_INSTALL_PREFIX}/lib/pkgconfig)
--- origsrc/libBigWig-0.4.9/libBigWig.pc.in	1970-01-01 00:00:00.000000000 +0000
+++ src/libBigWig-0.4.9/libBigWig.pc.in	2026-10-09 23:14:17.786930900 +0000
@@ -0,0 +1,10 @@
+prefix=@prefix@
+exec_prefix=@exec_prefix@
+libdir=@libdir@
+includedir=@includedir@
+
+Name: libBigWig
+Description: C library for handling bigWig files
+Version: @VERSION@
+Libs: -L${libdir} -lBigWig
+Cflags: -I${includedir}
