--- origsrc/delly-2.7.0/Makefile	2026-10-05 14:07:21.000000000 +0000
+++ src/delly-2.7.0/Makefile	2026-10-08 00:52:49.767817200 +0000
@@ -20,7 +20,7 @@ bindir ?= $(exec_prefix)/bin
 # Flags
 CXX ?= g++
 CXXFLAGS += -std=c++17 -isystem ${HTSLIBINCDIR} -pedantic -W -Wall -Wno-unknown-pragmas -D__STDC_LIMIT_MACROS -fno-strict-aliasing -fpermissive -pthread
-LDFLAGS += -L${HTSLIBLIBDIR} -lboost_iostreams -lboost_filesystem -lboost_program_options -lboost_date_time -pthread
+LDFLAGS += -L${HTSLIBLIBDIR} -lboost_iostreams -lboost_filesystem -lboost_program_options -lboost_date_time -lboost_system -pthread
 
 # Boost location
 ifneq (${BOOSTINCDIR},)
